src/utils/view.test.js

```diff
@@ -334,11 +334,11 @@ describe('view', () => {
       });
 
       // Resulting transform should clamp to minimum scale and center
-      expect(transform).toEqual({
-        k: minScaleFocus,
-        x: -50,
-        y: 60,
-      });
+      expect(transform.k).toEqual(minScaleFocus);
+
+      // In this example offset is fractional
+      expect(transform.x).toBeCloseTo(-43.333);
+      expect(transform.y).toBeCloseTo(6.666);
```

Comment:
It is quite interesting to see the y values to change by almost 10 times from the original value - is this directly related to the change in the `minScaleX` value? 

 As with my comment in earlier PRs, it might be more clear if the values are represented and compared against a calculated value rather than a specified number that is subject to change whenever we alter any pre-set value, as well as allowing us to truly test the transform rather than going down the easy path of re-plugging numbers whenever a test fails :) WDYT?

Reply:
Yeah these tests are due to be improved, at the moment they are just sanity checks (hence the fixed constants) because the logic is complex - this will be split up later to allow cleaner testing, rather than on the combined result.